core/src/test/java/org/hibernate/ogm/backendtck/id/gen/CustomIdGeneratorTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.id.gen;
+
+import static org.hibernate.ogm.utils.OgmAssertions.assertThat;
+
+import java.util.List;
+
+import org.hibernate.id.IdentifierGenerator;
+import org.hibernate.ogm.utils.jpa.OgmJpaTestCase;
+
+import org.junit.Test;
+
+/**
+ * Test user defined {@link IdentifierGenerator}
+ */
+public class CustomIdGeneratorTest extends OgmJpaTestCase {
+
+	@Test
+	public void test() {
```

Comment:
not critical, but let's try to use a better name for this method. `testidFromCustomGenerator`?